### ISSUE TITLE
Images get cropped

### ISSUE THREAD

**Issue by ironhak** [kepano/obsidian-minimal#527]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

Last version

### Describe the bug
Hi everyone, I'm using minimal theme and I'm experiencing this "bug".
Here's in live-preview mode what I see:
![image](https://user-images.githubusercontent.com/62611612/211198490-28cc5b0c-270d-4dbe-9359-0286e72c2da2.png)
Everything's good, but as soon as I go to reading view:
![image](https://user-images.githubusercontent.com/62611612/211198503-9fc47300-e813-4650-89fe-e9c9f95a3d3d.png)
As you can see image border get cropped. 

Any solution to this? Thank's

### Debug info

SYSTEM INFO:
	Obsidian version: v1.1.9
	Installer version: v1.1.9
	Operating system: Darwin Kernel Version 21.6.0: Sat Jun 18 17:07:22 PDT 2022; root:xnu-8020.140.41~1/RELEASE_ARM64_T6000 21.6.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal v6.1.18
	Snippets enabled: 6
	Restricted mode: off
	Plugins installed: 24
	Plugins enabled: 21
		1: Advanced Tables v0.18.1
		2: Banners v1.3.3
		3: Calendar v1.5.10
		4: Clear Unused Images v1.0.6
		5: Paste URL into selection v1.7.0
		6: Note Refactor v1.7.1
		7: Natural Language Dates v0.6.1
		8: Minimal Theme Settings v6.1.1
		9: Media Extended v2.11.1
		10: Kindle Highlights v1.8.1
		11: Hotkeys for starred files and searches v0.0.4
		12: Image Toolkit v1.3.1
		13: Highlightr v1.2.2
		14: Hider v1.2.4
		15: Zoom v1.1.1
		16: Underline v1.2.1
		17: Templater v1.16.0
		18: Style Settings v0.4.12
		19: Chat View v1.5.0
		20: Dataview v0.5.52
		21: Contextual Typography v2.2.4

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by ironhak** [comment]
Found a solution, maybe @kepano can put this on a future update or explain me why that was not the default setting so I can learn more :) 
```
/*MINIMAL THEME FIX*/
.img-grid .markdown-preview-section>.el-embed-image>p>.internal-embed img {
    object-fit: contain;
}
```

Tahnk's again for this amazing theme :)

**Comment by ltroj** [comment]
Same problem here. Is this a confirmed bug yet?

**Comment by sjmgarnier** [comment]
I can confirm the bug and that the snippet provided by @ironhak corrects it.

**Comment by sangmort** [comment]
Same, ironically was coming here to post the solution but saw @ironhak beat me to it :)

**Comment by kepano** [comment]
This is fixed in 6.2.2 with new Style Settings options
https://github.com/kepano/obsidian-minimal/releases/tag/6.2.2
